spec/controllers/account/chrome_extension_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Account::ChromeExtensionController, type: :controller do
+  let!(:company) { create(:company) }
+  before(:each) { sign_in company.user }
+
+  describe 'GET #new' do
+    it 'renders the #new view' do
+      get :new
+      expect(response).to have_http_status(200)
+      expect(response).to render_template :new
+    end
+  end
+
+  describe 'POST #create' do
+    context 'with valid attributes' do
+      let!(:valid_page) { build(:page) }
+      it 'add page' do
+        expect do
+          post :create, params: { page: { name: valid_page.title,
```

Comment:
add a spec when you are sending source_url: '' and title: ''

Reply:
Model 'Page' hasn't any validation, so that spec would pass.
I think I should add validations, right?

Reply:
add validation that URL must be present

Reply:
and validation company_id must present